ui/apps/platform/src/Containers/Collections/CollectionResults.tsx

```diff
@@ -94,15 +94,22 @@ function CollectionResults({
         (page: number) => fetchMatchingDeployments(dryRunConfig, page, filterText, selected),
         [dryRunConfig, filterText, selected]
     );
-    const { data, fetchNextPage, resetPages, clearPages, isEndOfResults, isRefreshingResults } =
-        usePaginatedQuery(queryFn, 10, {
-            debounceRate: 800,
-            manualFetch: true,
-            dedupKeyFn: ({ id }) => id,
-            onError: (err) => {
-                setConfigError?.(parseConfigError(err));
-            },
-        });
+    const {
+        data,
+        fetchNextPage,
+        resetPages,
+        clearPages,
```

Comment:
Not a blocker to put these name in alphabetical order, since that is the pattern elsewhere in collections.